intro_to_R_tidyverse/intro_to_r.Rmd

```diff
@@ -0,0 +1,391 @@
+---
+title: "Intro to R"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2019**
+
+### This RMarkdown takes you through the basics of R  
+
+*For more comprehensive tutorials:*  
+  - For a tutorial on R, RStudio and RMarkdown: https://ismayc.github.io/rbasics-book/  
+  - [Datacamp](https://www.datacamp.com/) has lots of good tutorials for R and 
+    the intro ones are free: 
+  
+This html is made from an [RMarkdown file](https://rmarkdown.rstudio.com/articles_intro.html). 
+Rmd's are handy for times you'd like to have output that has the code along with
+the output from that code. 
+Rmds also have a lot of options and features for what that output looks like and 
+what it includes. 
+
+## The Basics  
+R remembers what you tell it and stores assigned values in it's environment. 
+In RStudio, you will see what R has in it's environment in the "environment" panel.
+Here let's assign the value of `1` to `x` using the assignment operator: `<-`.
+```{r Basic assignment}
+x <- 1
+```
+
+Now R will print out `1` if we tell it `x`.
+```{r Recall}
+x
+```
+
+Use the print function to more explicitly ask R to recall what x is
+```{r print}
+print(x)
+```
+
+R will ignore what you say (allowing you to make comments on your code) if you 
+put hashtags/pound signs.
+```{r Comments}
+# This is important because you should always have comments to explain your code
+```
+
+Can't remember what values you have assigned to things in your environment? 
+Use the function `ls` to list everything in the current environment. 
+Just like in math, 
+*functions* are things that take input, transform or otherwise do something with this input, and then give you output. 
+In R, functions take input through *arguments* within their parantheses. 
+If the arguments don't make sense, you will get an *error* that will try to tell 
+you why R can't do what you asked. 
+Use a `?` mark before a function name to bring up the help page for that function
+so you can see how it's supposed to be used. 
+Or just search for the function by name in the *help bar* in RStudio.
+```{r Environment}
+# It will tell us that our object "x" is in the environment
+ls()
+
```

Comment:
![screen shot 2019-02-01 at 10 58 49 am](https://user-images.githubusercontent.com/15315514/52134001-94d79780-2610-11e9-99c3-43e6ea03275b.png)

This is what it showed me when I opened up the files. It would be confusing to see all these variables which weren't assigned visibly prior to this.

Reply:
That's because I forgot to run it again after I cleaned out the environment from other things I was doing. I will make sure it doesn't have that in the final version.